radio/src/opentx.cpp

```diff
@@ -1728,8 +1728,17 @@ void copyTrimsToOffset(uint8_t ch)
   storageDirty(EE_MODEL);
 }
 
+
+const uint8_t _pwrOnOffDuration_lookup[] = {
```

Comment:
why divide by 2 ?

Reply:
Well, linearly it makes: 2, 1.333, 0.666, 0. Feels a bit weird.

Reply:
Or really 3, 2, 1, 0 ? But makes the 3s the default value, which is quite long.